vulkan/vulkan_utilities.cc

```diff
@@ -10,16 +10,24 @@
 
 namespace vulkan {
 
+static bool sValidationLayersEnabled = false;
```

Comment:
Let's stay away from static variables. This makes it hard to perform validation on a per Flutter application level basis. This flag can be store in `VulkanApplication` instead and queried when the layers are being setup.

Reply:
Done